### DISCUSSION TITLE
equavalent for dnsmasq server= address= options

### DISCUSSION THREAD

**Discussion by rezad1393** [DNSCrypt/dnscrypt-proxy#2308] [Q&A]
is there anyway that I can add a list of website that like this to dnscrypt?
address=/web.archive.org/192.168.1.1
server=/steampowered.com/8.8.8.8

right now I have to combine dnsmasq and dnscrypt for just this.

**Comment by lifenjoiner** [comment]
All functionalities are guided here: [wiki](https://github.com/DNSCrypt/dnscrypt-proxy/wiki), and detailed in the shipped example files.
Seems you need **Cloaking** and **Forwarding**, maybe **Filtering**.
The logic here is you should put different purpose rules in separate files.

**Reply by rezad1393** [reply]
> All functionalities are guided here: [wiki](https://github.com/DNSCrypt/dnscrypt-proxy/wiki), and detailed in the shipped example files. Seems you need **Cloaking** and **Forwarding**, maybe **Filtering**. The logic here is you should put different purpose rules in separate files.

in wiki it says that "  You can point to a cloaking file using the cloaking_rules key" 
can I point to a folder containing multiple one of those? or maybe multiple keys for multiple files?

**Reply by lifenjoiner** [reply]
No. It should be a single file.

**Reply by rezad1393** [reply]
> No. It should be a single file.

can I ask for support for folder.d support like dnsmasq ?

**Reply by lifenjoiner** [reply]
Personally, I think every software (should) has its own logic/philosophy.
I don't stand for any others.

**Reply by rezad1393** [reply]
yes, but this is a suggestion.
dnsmasq also was like this and then added support for folder.d loading.
